packages/core/src/tools/create-logger.js

```diff
@@ -89,66 +99,19 @@ const toStdout = (event, msg, data) => {
   }
 };
 
-const isSafeUrl = (value) => {
-  let url;
-  try {
-    url = new URL(value);
-  } catch (err) {
-    return false;
-  }
-  return !(url.username || url.password || url.search);
-};
-
-const makeSensitiveBank = (event, data) => {
+const buildSensitiveValues = (event, data) => {
   const bundle = event.bundle || {};
-
-  const bank = { ...bundle.authData, ...process.env };
-  const sensitiveValues = Object.entries(bank).reduce(
-    (values, [key, value]) => {
-      if (SENSITIVE_KEYS.includes(key) || !isSafeUrl(value)) {
-        return [...values, value];
-      }
-      // Allow uncensored if the value is a safe URL and the key is not in
-      // SENSITIVE_KEYS
-      return [...values];
-    },
-    []
-  );
-
-  const matcher = (key, value) => {
-    if (_.isString(value)) {
-      const lowerKey = key.toLowerCase();
-      return _.some(SENSITIVE_KEYS, (k) => lowerKey.indexOf(k) >= 0);
-    }
-    return false;
-  };
-
-  dataTools.recurseExtract(data, matcher).forEach((value) => {
-    sensitiveValues.push(value);
-  });
-
-  return _.reduce(
-    sensitiveValues,
-    (bank, val) => {
-      // keeps short values from spamming censor strings in logs, < 6 chars is not a proper secret
-      // see https://github.com/zapier/zapier-platform-core/issues/4#issuecomment-277855071
-      if (val && String(val).length > 5) {
-        const censored = hashing.snipify(val);
-        bank[val] = censored;
-        bank[encodeURIComponent(val)] = censored;
-        try {
-          bank[Buffer.from(String(val)).toString('base64')] = censored;
-        } catch (e) {
-          if (e.name !== 'TypeError') {
-            throw e;
-          }
-          // ignore; Buffer is semi-selective about what types it takes
-        }
-      }
-      return bank;
-    },
-    {}
+  const authData = Object.values(bundle.authData || {});
+  // for the most part, we should censor all the values from authData
+  // the exception is safe urls, which should be filtered out - we want those to be logged
+  const sensitiveAuthData = authData.filter(
```

Comment:
This logic keeps "safe urls" around. there's code in scrubber to "intelligently" pull sensitive data out of urls which we could use here too, but we're also fine to censor all non-safe urls. 

If we did that, it would pull `my-secret` out of `https://www.example.com/whatever?token=my-secret&foo=bar`. Then instead of censoring the whole url, it would be `https://www.example.com/whatever?token=:censored:3:asdf:&foo=bar`. But, it's also fine as is.